src/lib/autoLaunch/linuxAutoLaunch.ts

```diff
@@ -1,29 +1,44 @@
+import { AutoLaunchParams, ChildReturnData } from '../types'
+import * as native from '../native'
+import * as fs from "@tauri-apps/api/fs"
 
-module.exports = {
+const linuxAutoLaunch = {
```

Comment:
Can we have this as a class that implements an interface shared for all 3 platforms?

Reply:
The class that implements the cross platform functions is in native.ts but I agree that things could be organized in a better way.